src/FeatureExtractor/Mobilenet.js

```diff
@@ -294,33 +297,32 @@ class Mobilenet {
     return prediction[0];
   }
 
-  // Static Method: get top k classes for mobilenet
-  static async getTopKClasses(logits, topK, callback = () => {}) {
-    const values = await logits.data();
-    const valuesAndIndices = [];
-    for (let i = 0; i < values.length; i += 1) {
-      valuesAndIndices.push({ value: values[i], index: i });
-    }
-    valuesAndIndices.sort((a, b) => b.value - a.value);
-    const topkValues = new Float32Array(topK);
-
-    const topkIndices = new Int32Array(topK);
-    for (let i = 0; i < topK; i += 1) {
-      topkValues[i] = valuesAndIndices[i].value;
-      topkIndices[i] = valuesAndIndices[i].index;
-    }
-    const topClassesAndProbs = [];
-    for (let i = 0; i < topkIndices.length; i += 1) {
-      topClassesAndProbs.push({
-        className: IMAGENET_CLASSES[topkIndices[i]],
-        probability: topkValues[i],
+  async load(filesOrPath = null) {
```

Comment:
@cvalenzuela and I just discussed this in person but for posterity we decided that in addition to accepting files, we would also allow the load function to work like:

```javascript
model.load('model.json', 'model.weights.bin');
```

Or does it make more sense to take a path and look for files named a specific way (a la LSTM)?

```javascript
model.load('path/to/model/');
```

Reply:
I updated it to be consistent with what tf.js is using (ie: specifing a `model.json`) and added a callback function. So this works now:

```javascript
classifier.load('/model/model.json', () => console.log('Loaded!') );
// or
classifier.load('https://mymodel.com/model.json', () => console.log('Loaded!') );
// or
classifier.load(loadBtn.files, () => console.log('Loaded!'));
```

Besides, using `indexeddb://my-model-1` and `localstorage://my-model-1` should also work.